src/gui/guiScrollBar.cpp

```diff
@@ -315,6 +316,12 @@ void GUIScrollBar::setPageSize(const s32 &size)
 	setPos(scroll_pos);
 }
 
+void GUIScrollBar::setArrowsVisible(bool visible)
+{
+	up_button->setVisible(visible);
+	down_button->setVisible(visible);
```

Comment:
I think, it would be better to add a button visibility attribute (bool or enum).
At least set boarder to width, height or 0.